### ISSUE TITLE
Submission by round detail view: Grouped by listing: Backend

### ISSUE THREAD

**Issue by gibbsmaya** [OpenTechFund/opentech.fund#814]
Grouped Listing on left hand side of Submission by Round page

Acceptance criteria
- [x] Contains same items as the base Submissions by Round list view queryset (submissions in the selected round) without filters applied
- [x] Submissions are grouped by status
- [x] Number of submissions in each status are presented
- [x] If no submissions in round display message to state this
- [x] If there is an error in fetching this data, display a 'sorry' message
- [x] QA criteria satisfied

Design: https://projects.invisionapp.com/share/KZPIO1NENQJ#/screens/340971430

**QA criteria**
Dev:

- [x] checked feature meets acceptance criteria/conforms exactly to the specification.
- ~provided good unit test coverage (if this is non-trivial behaviour).~
- [x] checked all tests for the project pass with this feature enabled.
- [x] checked code conforms to the project coding standards.
- [x] had code reviewed by another developer and resolved any issues raised.
- [x] tested this feature as an end user of the website/app (Can I get to it? Is it useable? Can I break it? Does it work in an end-to-end context?)
- [x] checked that this feature works on the server/s I am deploying it to.

QA:

 * [x] tested this feature as a front end user and it meets the acceptance criteria/conforms to the specification and design.
 * [x] checked that the feature works on the server/s deployed to

**Comment by tm-kn** [comment]
Example - https://dev-apply.opentech.fund/apply/api/submissions/rounds/732/

**Comment by gibbsmaya** [comment]
Remaining tasks are to fetch the round details from the API which will include the "friendly" status names and the ordering of statuses within the round. Will split this out into a separate task for when this is available via the API.

Other than this this looks good!

Assigning to @todd-dembrey for deployment to test.

**Comment by todd-dembrey** [comment]
@gibbsmaya This is now deployed on test.

**Comment by gibbsmaya** [comment]
Hi @fourthletter, @danblah

This work is now ready for testing. This work is on the new submission by round detail view which an example of can be found here https://test-apply.opentech.fund/apply/submissions/rounds/732/ and using the switcher in the top right to switch to the detail view page rather than the list view. Note that this ticket is not the entire functionality of this page: we are looking specifically at the left hand column of this page. The remainder of this page is being developed partly in the current sprint and partly in the next couple of sprint. So we will be adding to this as time goes on.

Please test this against the acceptance criteria noted in the initial issue description where you will also see a link to the design. Note that the status names are not very human-friendly, they are in fact the machine names of the statuses. This is because these are currently not coming from the API (as the API is not yet complete) and this is due to be updated in the next sprint as per #844. Please also note that the acceptance criteria for the front end of this (how it looks) is captured in a separate issue #813 which is also with you for testing so worth testing these in parallel.

Please let us know if anything is unclear or if you have any questions. Otherwise please provide feedback of any issues you find or if everything is working as expected, please confirm and we can get this work deployed to the live site and issue closed off.

Thanks,
Maya

**Comment by danblah** [comment]
Looks good, acceptance criteria met from my testing.

**Comment by gibbsmaya** [comment]
Thanks @danblah . Assigning to @todd-dembrey to arrange deployment to production.
